h2o-core/src/test/java/water/fvec/VecTest.java

```diff
@@ -197,26 +197,16 @@ public void map(Chunk c) {
   }
 
   @Test public void testChunkIdxWithDeletedVec() {
+    Key<Vec> k = null;
     try {
       Scope.enter();
       Vec v = Scope.track(makeConN((long) 1e6, 16));
-      new MRTask() {
-        @Override
-        protected void setupLocal() {
-          Vec v = _fr.anyVec();
-          if (v != null) {
-            v.remove(); // evil happens here
-          }
-        }
-        @Override
-        public void map(Chunk c) {
-          if (c.vec().get_type() != Vec.T_NUM)
-            throw new IllegalStateException("Expected a numeric Vec");
-        }
-      }.doAll(v);
+      k = v._key;
+      v.remove();
+      v.chunkForChunkIdx(0);
```

Comment:
if you expect an exception, you should probably add a `fail()` statement before `catch`.